rust/rebase/src/flow/email.rs

```diff
@@ -130,25 +136,34 @@ impl Flow<Ctnt, Stmt, Prf> for SendGridBasic {
             ));
         }
 
+        let challenge_vec: Vec<&str> = proof.challenge.split(&self.challenge_delimiter).collect();
+        if challenge_vec.len() != 2 {
+            return Err(FlowError::Validation(
+                "Challenge in unexpected formatt".to_string(),
```

Comment:
```suggestion
                "Challenge in unexpected format".to_string(),
```
total nitpick :sweat_smile: